### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by afs25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+### Processing
+1. Identifies the target environment of the appointment event received from Acuity
+2. Forwards the event to the AWS account containing target environment, according to configuration in the "ForwardingMap" table
+
+## Interfaces
+### Events raised
+| Bus                      | Destination(s) | Event(s)                     | Description                                                           |
+|--------------------------|----------------|------------------------------|-----------------------------------------------------------------------|
+| Thiscovery               | acuity-router  | raw_acuity_event          | Event containing the payload of the API call initiated by Acuity      |
+| Thiscovery               | acuity-router  | processed_acuity_event          | Processed Acuity event, with target account and environments resolved |
```
Comment (sophiemachin) [inline README.md:24]: PyCharm was moaning this table wasn't properly formatted, it suggested:

| Bus        | Destination(s) | Event(s)               | Description                                                           |
|------------|----------------|------------------------|-----------------------------------------------------------------------|
| Thiscovery | acuity-router  | raw_acuity_event       | Event containing the payload of the API call initiated by Acuity      |
| Thiscovery | acuity-router  | processed_acuity_event | Processed Acuity event, with target account and environments resolved |

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+### Events consumed
+| Bus                      | Source(s)                    | Event(s)                     | Description                                            |
+|--------------------------|------------------------------|------------------------------|--------------------------------------------------------|
+| Thiscovery               | acuity-router  | raw_acuity_event          | Triggers processing of the data received from Acuity   |
+| Thiscovery               | acuity-router  | processed_acuity_event          | Triggers forwarding of event to the target AWS account |
```
Comment (sophiemachin) [inline README.md:29]: Same for this table

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+forwards those events to the thiscovery event bus of the 
+destination account.
+
+## Responsibilities
+### Data storage
+1. Dynamodb "ForwardingMap" table (Stores configuration)
+
+### Processing
+1. Identifies the target environment of the appointment event received from Acuity
+2. Forwards the event to the AWS account containing target environment, according to configuration in the "ForwardingMap" table
```
Comment (sophiemachin) [inline README.md:17]: There is inconsistency for line lengths - is this deliberate? Some look like they are capped at 80 and others (like this one) don't have any restriction

**Review by sophiemachin — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,2 +1,38 @@
...
-Thiscovery router service for Acuity webhook calls
+## Purpose
+Allows multiple thiscovery environments (e.g. test, staging, prod)
+to use a single Acuity account without causing 
+clashes/interference. 
+
+Listens to incoming requests posted by Acuity webhooks and 
+forwards those events to the thiscovery event bus of the 
+destination account.
+
```
Comment (this-andy) [inline README.md:10]: Is it not also the case that this stack also decides if Acuity webhook events need to be sent to VIEWS or whether they are non-VIEWS related and can therefore be forwarded to other Thiscovery microservices?

File: README.md (hunk 3 above)
Comment (afs25) [reply README.md:17]: In lists like this one, PyCharm adds a new item every time I hit return to break long lines. That's why I ended up not breaking long lines in lists.

File: README.md (hunk 4 above)
Comment (afs25) [reply README.md:10]: In the context of this router service, VIEWS is a thiscovery microservice. There is nothing special about it that requires different processing.

**Pull request merged**